internal/functions/serve/templates/main.ts

```diff
@@ -156,7 +156,8 @@ Deno.serve({
     console.error(`serving the request with ${servicePath}`);
 
     // Ref: https://supabase.com/docs/guides/functions/limits
-    const memoryLimitMb = 256;
+    const computeMultiplier = Math.max(Math.min(parseFloat(functionsConfig[functionName].computeMultiplier ?? '1'), 4), 1)
```

Comment:
```suggestion
    const computeMultiplier = Math.max(Math.min(functionsConfig[functionName].computeMultiplier ?? 1, 4), 1)
```

No need to parse from string because serialised json already sets this field as number.

Should also update the type definition for `functionConfig` https://github.com/supabase/cli/pull/2912/files#diff-44113b100b5a3840b42b5d08768b70a8f82055f9a7d11b37f31e2ea72d51e50dR52

```
computeMultiplier?: number
```

Optionally, consider emitting a log message when this value is clamped so users are aware of this limit when serving locally, ie. 1 <= x <=4. Otherwise the only way to know is via the api error when deploying.